pkg/operator/encryption/controllers.go

```diff
@@ -35,6 +38,7 @@ func NewControllers(
 	kubeInformersForNamespaces operatorv1helpers.KubeInformersForNamespaces,
 	secretsClient corev1.SecretsGetter,
 	eventRecorder events.Recorder,
+	resourceSyncer resourcesynccontroller.ResourceSyncer,
```

Comment:
could we pass the concrete obj not an interface so that we don't have to change the `ResourceSyncer` interface?

Reply:
Done